src/main/java/org/terasology/climateConditions/HypothermiaSystem.java

```diff
@@ -48,7 +48,7 @@ public void observeDangerZone(MovedEvent event, EntityRef player) {
         }
     }
 
-    @ReceiveEvent(components = {HypothermiaComponent.class})
+    @ReceiveEvent
```

Comment:
Was that removed by accident? I think now the player is slowed down all the time o.O
```suggestion
    @ReceiveEvent(components = {HypothermiaComponent.class})
```

Reply:
Yes, while I was changing all this, I messed up my workspace, so I commited and pushed everything without checking. Sorry for that 😅.